app/views/products/_product.html.erb

```diff
@@ -1,6 +1,14 @@
 <tr id="<%= dom_id product %>" class="<%= cycle('list_line_odd', 'list_line_even') %>">
   <td class="description">
     <h1><%= product.title %></h1>
+    <div>Users Ratings: <%= average_rating(product) %></div>
+
+    <%= form_with url: ratings_path, method: :post, local: false do |form| %>
+      <%= form.hidden_field :product_id, value: product.id %>
+      <%= form.select :rating, generate_ratings, selected: current_product_rating_by_user(product.ratings, @logged_in_user) %>
+      <%= form.submit 'Submit Rating' %>
```

Comment:
are we showing errors from the response?

Reply:
Search about local: true responses